src/components/popover/popover.js

```diff
@@ -47,7 +47,7 @@ export const BPopover = /*#__PURE__*/ Vue.extend({
       default: () => getComponentConfig(NAME, 'boundary')
     },
     boundaryPadding: {
-      type: [Number, String],
+      type: Number,
```

Comment:
We should allow for Number as strings here, so users don't need to v-bind when passing a number.